src/components/TableBody.js

```diff
@@ -240,6 +240,11 @@ class TableBody extends React.Component {
                     fixedHeader={options.fixedHeader}
                     checked={this.isRowSelected(dataIndex)}
                     expandableOn={options.expandableRows}
+                    // When rows are expandable and selectable, but this row isn't expandable, set this to enabled.
+                    // This will add a new class, MUIDataTableSelectCell-expandDisabled and can be overridden when wanted.
+                    hideExpandButton={
+                      !this.isRowExpandable(dataIndex) && options.expandableRows && options.selectableRows
```

Comment:
Hm, shouldn't this option be independent of what's happening with `selectableRows`? I would expect an expand disabled class to only matter given expandable options. For example, we could have this situation occur even when selectable rows are turned off entirely in the options.

Reply:
ooops yeah, I think this is my bad. Not sure what went to my head. I will remove the `selectableRows` clause.